### PR TITLE
DRIVERS-600 Add zstd as compression algorithm

### PR DESCRIPTION
**Pull request by xdg**
Apparently, we never updated OP_COMPRESSED before opening DRIVERS-600.

### CODE REVIEW

**Comment by xdg** [comment]
It's not a very well written spec so I was hoping to avoid a rewrite, but if you think it needs it, I'll take a cut at it.

**Comment by jyemin** [comment]
I think a few extra sentences would reduce confusion.

**Comment by xdg** [comment]
@jyemin PTAL

File: source/compression/OP_COMPRESSED.rst [hunk 1]
```diff
@@ -44,8 +44,8 @@ interpreted as described in `RFC 2119 <https://www.ietf.org/rfc/rfc2119.txt>`_.
 Terms
 =====
 
-*Compressor* - A compression algorithm, such as snappy or zlib.
-
+*Compressor* - A compression algorithm.  There are currently three supported
+algorithms: snappy, zlib and zstd.
```
Comment (jmikola) [inline source/compression/OP_COMPRESSED.rst]: Can we throw an [Oxford comma](https://i.kym-cdn.com/entries/icons/original/000/017/771/the-oxford-comma_52c855ed979ed_w1500.jpg) in this list?

**Comment by xdg** [comment]
Rebased and merged as 0d08743afe1379b0db4c614a7837ee507910dfe4

**Pull request closed without merging**